.circleci/config.yml

```diff
@@ -0,0 +1,67 @@
+docker-postgres: &postgres
+  image: circleci/postgres:9.6-alpine-ram
+  environment:
+    POSTGRES_USER: ubuntu
+    POSTGRES_PASSWORD: ""
+    POSTGRES_DB: snap
+
+docker-stack: &haskell
+  image: fpco/stack-build:lts-11.18
+  environment:
+    HOST: localhost
+    USER: ubuntu
+    PORT: 5432
+    PASS: ""
+    DB: snap
+    PGHOST: localhost
+    PGUSER: ubuntu
+    PGPASS: ""
+    PGDATABASE: snap
+
+version: 2
+jobs:
+  project:
+    docker:
+      - <<: *haskell
+      - <<: *postgres
```

Comment:
Do you know what this means?

Reply:
If I am not wrong, does it mean to use the containers named "haskell" and "postgres" defined in the start of `config.yml`

Reply:
Actually they are like variables of sorts... why are you referencing a variable if you are not using it in more than one place?